### ISSUE TITLE
Disable to log "I'm ready!"

### ISSUE THREAD

**Issue by hata6502** [codex-team/editor.js#962]
The message when Editor.js is ready: 

I'm ready! (ﾉ◕ヮ◕)ﾉ*:･ﾟ✧

may be not needed for some production uses. 
Also this message may prevent to use test tools. 
When using jest to application contains Editor.js, "I'm ready!" messages are logged many times. 

```
      %cEditor.js 2.15.1%c I'm ready! (ﾉ◕ヮ◕)ﾉ*:･ﾟ✧ line-height: 1em;
                  color: #006FEA;
                  display: inline-block;
                  font-size: 11px;
                  line-height: 1em;
                  background-color: #fff;
                  padding: 4px 9px;
                  border-radius: 30px;
                  border: 1px solid rgba(56, 138, 229, 0.16);
                  margin: 4px 5px 4px 0; color: #E24A75
      %cEditor.js 2.15.1%c I'm ready! (ﾉ◕ヮ◕)ﾉ*:･ﾟ✧ line-height: 1em;
                  color: #006FEA;
                  display: inline-block;
                  font-size: 11px;
                  line-height: 1em;
                  background-color: #fff;
                  padding: 4px 9px;
                  border-radius: 30px;
                  border: 1px solid rgba(56, 138, 229, 0.16);
                  margin: 4px 5px 4px 0; color: #E24A75
……
```

https://github.com/codex-team/editor.js/blob/master/src/components/core.ts#L78

**Comment by hata6502** [comment]
Thank you!!
It became easier to test!

**Comment by neSpecc** [comment]
Resolved in 2.16

**Comment by zbcms** [comment]
config 
const LogLevels = {
    VERBOSE: 'VERBOSE',
    INFO: 'INFO',
    WARN: 'WARN',
    ERROR: 'ERROR',
}
set to config
logLevel={LogLevels.ERROR}
